src/Simulation/Core/RuntimeMetadata.cs

```diff
@@ -42,12 +42,16 @@ public class RuntimeMetadata
         /// <summary>
         /// True if operation is composed of multiple operations.
         /// </summary>
-        /// </summary>
         /// <remarks>
         /// This is used in composite operations, such as <c>ApplyToEach</c>.
         /// </remarks>
         public bool IsComposite { get; set; }
 
+        /// <summary>
+        /// True if operation is a classically-controlled conditional operation.
+        /// </summary>
+        public bool IsConditional { get; set; }
```

Comment:
I'm not sure I like this name/description for this.

Reply:
Is the name not descriptive enough or is it the description that is not clear?
This flag is `true` when the operation is an If-Else statement, like `ApplyIfElseR` and will contain 2 `Children` (one for each measurement value).